resources/create_modulefile.rb

```diff
@@ -0,0 +1,34 @@
+resource_name :create_modulefile
+provides :create_modulefile
+
+property :source_path, String, name_property: true
```

Comment:
I think you should be fine just by removing this`name_property` from this parameter